wiki/Tournaments/4DM/2024/en.md

```diff
@@ -60,10 +60,57 @@ The 4 Digit osu!mania World Cup 2024 is run by various community members.
 - [Player registrations](http://4dm-registrations.dannypx.com/)
 - [Staff applications](https://forms.gle/ob5i22KF3GjDMnai6)
 - [Livestream](https://www.twitch.tv/4digitmwc)
+- [Pick'em predictions website](https://pickem.hwc.hr/tournaments/134)
 - [Tournament bracket](https://challonge.com/4DM2024)
 - Spreadsheets
   - **[Master](https://docs.google.com/spreadsheets/d/1y8w8SkP95VDyjqo4-APiCOBP27HcG3nIcKyb8Cqftk4/edit?rm=minimal)**
 
+## Participants
+
+|  | Country | Members |
+| :-: | :-- | :-- |
+| ::{ flag=AR }:: | **Argentina** | **[APRL03](https://osu.ppy.sh/users/11394892)**, [joako_loco](https://osu.ppy.sh/users/13659177), [monten505](https://osu.ppy.sh/users/21702734), [Mateo12345](https://osu.ppy.sh/users/16481146), [Iplaygames13](https://osu.ppy.sh/users/24253806), [ElMancoDelSiglo](https://osu.ppy.sh/users/22447212) |
+| ::{ flag=AU }:: | **Australia** | **[PotassiumF](https://osu.ppy.sh/users/4247722)**, [HD_AdreNaline](https://osu.ppy.sh/users/10540368), [Orcanos](https://osu.ppy.sh/users/13762441), [-Ainsel-](https://osu.ppy.sh/users/27855382), [Jerold](https://osu.ppy.sh/users/18481219), [-Willow-](https://osu.ppy.sh/users/11247298) |
+| ::{ flag=BR }:: | **Brazil** | **[Namirin-chan](https://osu.ppy.sh/users/6761832)**, [zKuri](https://osu.ppy.sh/users/25445118), [MORDEST68](https://osu.ppy.sh/users/28527409), [Villanovinho](https://osu.ppy.sh/users/29512763), [1Nuttelinha1](https://osu.ppy.sh/users/13104856), [soutin](https://osu.ppy.sh/users/11771029) |
+| ::{ flag=CA }:: | **Canada** | **[Holo the Wise](https://osu.ppy.sh/users/17036270)**, [CommandoBlack](https://osu.ppy.sh/users/7025841), [Robert6400](https://osu.ppy.sh/users/11467559), [twitch chat](https://osu.ppy.sh/users/12723207), [dylan52528](https://osu.ppy.sh/users/13145344), [Icetiger333](https://osu.ppy.sh/users/14461963) |
+| ::{ flag=CB }:: | **Caribbean** | **[JordqnPlayz](https://osu.ppy.sh/users/11980833)**, [Suwano](https://osu.ppy.sh/users/16274535), [TheBSGovernment](https://osu.ppy.sh/users/29575916) |
```

Comment:
https://osu1.roanh.dev/wiki/en/Tournaments/4DM/2024#participants

![image](https://github.com/ppy/osu-wiki/assets/36758269/eff94c33-0664-459f-8195-e87c3167ebf3)

Reply:
Aaah, yes, allow me to add the flag real quick. Thanks for noticing.